cypress/integration/user.test.js

```diff
@@ -64,7 +64,7 @@ describe('User list and items test', () => {
           .get("button[aria-label='Редагувати']:first")
           .click()
           .wait(2000)
-          .get('.makeStyles-userName-26')
+          .get('#name')
```

Comment:
Please, don't use and `id` use `data-cy` attribute on components and then query it from cypress tests

Reply:
fixed